doc/developer/cli.rst

```diff
@@ -106,6 +106,18 @@ convention. Please do not scatter individual CLI commands in the middle of
 source files; instead expose the necessary functions in a header and place the
 command definition in a ``*_vty.[ch]`` file.
 
+JSON Operational State
+^^^^^^^^^^^^^^^^^^^^^^
+When adding a CLI command that will produce JSON it is customary to add a
+keyword ``json`` to end the command string, e.g.::
+
+  show ip ospf json
+
+When adding new JSON definitions to FRR, first search for any existing standard
```

Comment:
It's great to be making this point - but the other place where we talk about json keys is in the 'workflow' doc. Should this note be there or should the json info there be consolidated into the 'cli' doc?

Reply:
I am for `camelCase` only. No underscores or dashes. Also, we should somehow fix the JSON key naming for time fields, because now it very sucks, having blablaTimerSec, blablaTimerMsec, blablaMsec, blablaMsec, blablaSeconds, blablaTimerRemaining, etc. Just random naming.

Reply:
As per the discussion in the development meeting, I've submitted new text. This documents that JSON needs YANG model backing even if that means writing it, and that the resulting YANG kebab-case should be converted to camelCase for the JSON output.